zap/src/main/java/org/zaproxy/zap/view/AboutDialog.java

```diff
@@ -66,22 +66,24 @@ public AboutDialog(Frame owner, boolean modal) {
         gbcPanel.anchor = GridBagConstraints.NORTHWEST;
         gbcPanel.weightx = 1.0D;
         gbcPanel.weighty = 1.0D;
-        gbcPanel.ipady = 2;
+        gbcPanel.ipady = 50;
         gbcPanel.gridwidth = 2;
 
         gbcButtons.gridx = 1;
         gbcButtons.gridy = 1;
         gbcButtons.insets = new Insets(2, 2, 2, 2);
         gbcButtons.anchor = GridBagConstraints.SOUTHEAST;
 
-        mainPanel.add(new AboutPanel(), gbcPanel);
```

Comment:
Revert?

Reply:
Done. Sorry at some point I'd been trying to set something with the panel before adding it :wink: